lib/msf/core/opt.rb

```diff
@@ -26,7 +26,7 @@ def self.CPORT(default=nil, required=false, desc="The local client port")
     end
 
     # @return [OptAddress]
-    def self.LHOST(default=nil, required=true, desc="The listen address")
+    def self.LHOST(default=nil, required=false, desc="The listen address")
```

Comment:
any better workaround for this? will this break things?

Reply:
I broke tests that are based in it being "true". :trollface: 

When doing anything that will impact core code like this please be sure to migrate a test database with `RAILS_ENV=test rake db:create db:migrate` and run `rake spec` to determine what tests might depend on known state.